### PR TITLE
Add support for the `concordant` mode test in Fisher's and Stouffer's estimators

### PR DESCRIPTION
**Pull request by JulioAPeraza**
Closes None.

Changes proposed in this pull request:

- Currently, we only support `directed` tests in Fisher's and Stouffer's estimators. The new parameter `two_sided` (following Nilearn's naming convention) adds support for the `concordant` (two-sided) test.
- That parameter will be used in the Diagnostics module to determine if the diagnostic should be run for z < 0.

### CODE REVIEW

**Comment by JulioAPeraza** [comment]
It looks like something changed in Nilearn's `plot_stat_map`. The color bars for maps with only positive values in the example gallery are not symmetric by default: https://nimare--884.org.readthedocs.build/en/884/auto_examples/02_meta-analyses/01_plot_cbma.html#sphx-glr-auto-examples-02-meta-analyses-01-plot-cbma-py

**Comment by JulioAPeraza** [comment]
This could also be addressed using sequential colormaps instead. I open an issue [here](https://github.com/nilearn/nilearn/issues/4421) to see if there is a better way to approach this.

**Comment by JulioAPeraza** [comment]
When I checked the gallery the other day, it looked like the RTD was not properly built. Setting `symmetric_cbar` to `True` in `plot_stat_map` actually solved the issue.

File: nimare/resources/references.bib [hunk 1]
```diff
@@ -529,3 +529,16 @@ @article{geoffroy2001poisson
   year={2001},
   publisher={Taylor \& Francis}
 }
+
+@article{winkler2016non,
```
Comment (jdkent) [inline nimare/resources/references.bib:533]: is this referenced in the permutedOLS class?

File: nimare/meta/ibma.py [hunk 2]
```diff
@@ -337,12 +358,15 @@ class Stouffers(IBMAEstimator):
 
     _required_inputs = {"z_maps": ("image", "z")}
 
-    def __init__(self, use_sample_size=False, **kwargs):
+    def __init__(self, use_sample_size=False, two_sided=True, **kwargs):
```
Comment (jdkent) [inline nimare/meta/ibma.py:361]: this changes the default to concordant right? we should make note that this is different from previous behavior (but I do agree with changing this default behavior)

File: nimare/resources/references.bib (hunk 1 above)
Comment (JulioAPeraza) [reply nimare/resources/references.bib:533]: Good catch! I added that when I defined the `mode` parameter, but later, I removed it in favor of `two_sided` and forgot to remove the citation.

**Comment by JulioAPeraza** [comment]
@jdkent Thanks for the review! I added a note for the change in the `versionchanged` field of the docstring. We could also include a sentence in the release note of 0.2.3.

**Review by jdkent — APPROVED** [review]
LGTM, thanks @JulioAPeraza!

**Pull request merged**